director/commands/init.py

```diff
@@ -20,6 +20,8 @@
 DIRECTOR_ENABLE_DARK_THEME=false
 DIRECTOR_ENABLE_HISTORY_MODE=false
 DIRECTOR_REFRESH_INTERVAL=30000
+DIRECTOR_REPO_LINK="https://github.com/ovh/celery-director"
```

Comment:
Same comment as `self.REPO_LINK` in the `settings.py` file.

Reply:
see  answer to https://github.com/ovh/celery-director/pull/141/files/3d2c0bfcac90c9740b84d34fb3514fd3f175951b#r849687363